lib/if.h

```diff
@@ -195,7 +195,7 @@ struct if_link_params {
 	uint32_t min_delay;		/* Link Min Delay */
 	uint32_t max_delay;		/* Link Max Delay */
 	uint32_t delay_var;		/* Link Delay Variation */
-	float pkt_loss;			/* Link Packet Loss */
+	uint32_t pkt_loss;		/* Link Packet Loss */
```

Comment:
So it looks like you are fixing a bug here. Previously the code stored the straight percentage in pkt_loss but it is used as if a a count of .000003s.

Reply:
Correct.